### PR TITLE
Added 5/11/2017 meeting notes

### PR DESCRIPTION
**Pull request by nebrius**
Added meeting notes for 5/11. Apologies for the delay everyone!

### CODE REVIEW

File: meetings/2017-11-05.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+* **Minutes Google Doc**: https://docs.google.com/document/d/1urhinDNJK-EEMziNob8-mlsOEZM_zecluDIgBA6kVOw/
+* _Previous Minutes Google Doc_: https://docs.google.com/document/d/1YevHuA_csoTltDJ6mHr21fetZshNqWpsRbBRkSHhmYE/
+## Present
+- Ashley Williams ([ashleygwilliams](https://github.com/ashleygwilliams))
+- Bryan Hughes ([nebrius](https://github.com/nebrius))
+- Gregor Martynus ([gr2m](https://github.com/gr2m))
+- Tracy Hinds ([hackygolucky](https://github.com/hackygolucky))
+- William Kapke ([williamkapke](https://github.com/williamkapke))
+- Tierney Coren ([bnb](https://github.com/bnb))
+- Michael Dawson
```
Comment (mhdawson) [inline meetings/2017-11-05.md]: Michael Dawson ([mhdawson](https://github.com/mhdawson)

File: meetings/2017-11-05.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+* **GitHub Issue**: [TSC#227](https://github.com/nodejs/TSC/issues/227)
+* **Minutes Google Doc**: https://docs.google.com/document/d/1urhinDNJK-EEMziNob8-mlsOEZM_zecluDIgBA6kVOw/
+* _Previous Minutes Google Doc_: https://docs.google.com/document/d/1YevHuA_csoTltDJ6mHr21fetZshNqWpsRbBRkSHhmYE/
+## Present
+- Ashley Williams ([ashleygwilliams](https://github.com/ashleygwilliams))
+- Bryan Hughes ([nebrius](https://github.com/nebrius))
+- Gregor Martynus ([gr2m](https://github.com/gr2m))
+- Tracy Hinds ([hackygolucky](https://github.com/hackygolucky))
+- William Kapke ([williamkapke](https://github.com/williamkapke))
+- Tierney Coren ([bnb](https://github.com/bnb))
```
Comment (bnb) [inline meetings/2017-11-05.md]: `Tierney Coren` > `Tierney Cyren` ❤️

File: meetings/2017-11-05.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+- Michael Dawson 
+- Penny
```
Comment (bnb) [inline meetings/2017-11-05.md]: In addition to @mhdawson's comment, add:
`Penny ([pup](https://github.com/pup))`

Comment (pup) [inline meetings/2017-11-05.md]: oh yeah hello

**Comment by nebrius** [comment]
Cleaned up the attendee list, PTAL.

**Pull request merged**